docs/modules/druid/pages/usage-guide/resources-and-storage.adoc

```diff
@@ -8,25 +8,68 @@ Druid uses S3 or HDFS for deep storage, so no extra https://kubernetes.io/docs/c
 
 include::home:concepts:stackable_resource_requests.adoc[]
 
-If no resources are configured explicitly, the Druid operator uses the following defaults:
+=== Resource Requests
+
+include::home:concepts:stackable_resource_requests.adoc[]
+
+A minimal HA setup consisting of 2 Pods of each role has the following https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[resource requirements]:
+
+* `4700m` CPU request
+* `13800m` CPU limit
+* `11376m` memory request and limit
+
+Of course, additional services, require additional resources. For Stackable components, see the corresponding documentation on further resource requirements.
+
+Corresponding to the values above, the operator uses the following resource defaults:
 
 [source,yaml]
 ----
-brokers:
-  roleGroups:
-    default:
-      config:
-        resources:
-          cpu:
-            min: '200m'
-            max: "4"
-          memory:
-            limit: '2Gi'
+spec:
+  brokers:
+    config:
+      resources:
+        cpu:
+          min: 100m
+          max: 400m
+        memory:
+          limit: 512Mi
+  routers:
+    config:
+      resources:
+        cpu:
+          min: 100m
+          max: 400m
+        memory:
+          limit: 512Mi
+  historical:
+    config:
+      resources:
+        cpu:
+          min: 300m
+          max: 1200m
+        memory:
+          limit: 1.5Gi
```

Comment:
In the other operators we have used 1500Mi, would leave it up to you to decide
Applies to all the code below as well